automated install/basic-install.sh

```diff
@@ -1490,17 +1479,21 @@ disable_dnsmasq () {
       echo -e "${OVER}  ${INFO} Checking if dnsmasq is still running"
       silent53check
         if [[ $dnsmasq_flag = "true" ]]; then
-          echo -e "${OVER}  ${EXCL} dnsmasq still active, this is most likely due to the fact that ${COL_LIGHT_RED}dnsmasq
-      was loaded via a non convetional method. This might cause future conflicts with FTLDNS${COL_NC}"
-          ${SUDO} pkill dnsmasq
-          echo -e "${OVER}  ${TICK} dnsmasq process killed"
+          kill_dnsmasq
         fi
       fi
   else
     echo -e "${INFO} dnsmasq is not enabled"
   fi
 }
 
+kill_dnsmasq(){
+echo -e "${OVER}  ${EXCL} dnsmasq still active, this is most likely due to the fact that ${COL_LIGHT_RED}dnsmasq
+      was loaded via a non convetional method. This might cause future conflicts with FTLDNS${COL_NC}"
+${SUDO} pkill dnsmasq
+echo -e "${OVER}  ${TICK} dnsmasq process killed"
+
+}
```

Comment:
When I said move it out of the if statement, I meant just end the if statement earlier (you don't need to make it into a function):
```bash
if [[ $condition1 ]]; then
    # unique stuff #1
    # duplicated stuff
else
    # unique stuff #2
    # duplicated stuff
fi
```
vs
```bash
if [[ $condition1 ]]; then
    # unique stuff #1
else
    # unique stuff #2
fi
# duplicated stuff
```